### DISCUSSION TITLE
Email notification when a question is asked by a student.

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#23] [Feature requests]
Dear, 

As I don't check constantly for new questions in the different courses, I ask my students to send me a seperate email to inform me. It would be nice to receive an email notification from Dodone (maybe with a parameter to switch it off for those who don't won't this) so I can react faster to asked questions. 

Rgds, 
Marco Thijs

**Comment by niknetniko** [comment]
There is an (older) issue you can follow where some ideas about emails for questions are collected: https://github.com/dodona-edu/dodona/issues/2355.
